sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/BlobQueryReader.java

```diff
@@ -323,13 +323,12 @@ private static JsonTextConfiguration transformJson(BlobQueryJsonSerialization js
      * @param parquetSerialization {@link BlobQueryParquetSerialization}
      * @return {@link JsonTextConfiguration}
      */
-    // TODO (gapra): uncomment when parquet is released
-    /*private static Object transformParquet(BlobQueryParquetSerialization parquetSerialization) {
+    private static Object transformParquet(BlobQueryParquetSerialization parquetSerialization) {
```

Comment:
Mind adding more context to this method on why it just returns a `new Object()`.